sample-queries/sample-queries.json

```diff
@@ -1845,6 +1845,204 @@
+      "skipTest": false
+    },
+    {
+      "id": "97cb3895-df00-4b7d-bf25-2b3d75fc8da0",
+      "category": "Security",
+      "method": "GET",
+      "humanName": "get indicators for an article",
+      "requestUrl": "/beta/security/threatIntelligence/articles/{articleId}/indicators",
+      "docLink": "https://github.com/microsoftgraph/microsoft-graph-docs/pull/20472/files?short_path=bbfa479",
+      "skipTest": false
+    },
+    {
+      "id": "e98b1987-8104-4cca-8954-99038e6a25bf",
+      "category": "Security",
+      "method": "GET",
+      "humanName": "get an article indicator",
+      "requestUrl": "/beta/security/threatIntelligence/articleIndicators/{articleIndicatorId}",
+      "docLink": "https://github.com/microsoftgraph/microsoft-graph-docs/pull/20472/files?short_path=bbfa479",
+      "skipTest": false
+    },
+    {
+      "id": "1e5b37e9-b330-4cbc-9cd3-b2fc9dcd104c",
+      "category": "Security",
+      "method": "GET",
+      "humanName": "get a host",
+      "requestUrl": "/beta/security/threatIntelligence/hosts/{hostId}",
+      "docLink": "https://github.com/microsoftgraph/microsoft-graph-docs/pull/20472/files?short_path=bbfa479",
+      "skipTest": false
+    },
+    {
+      "id": "f135e0cd-f8f3-4ed9-baf4-09663a6835dd",
+      "category": "Security",
+      "method": "GET",
+      "humanName": "get the reputation for a host",
+      "requestUrl": "/beta/security/threatIntelligence/hosts/{hostId}/reputation",
+      "docLink": "https://github.com/microsoftgraph/microsoft-graph-docs/pull/20472/files?short_path=bbfa479",
+      "skipTest": false
+    },
+    {
+      "id": "ecf9181b-b89a-47a2-bb8c-43ad40e4d0cf",
+      "category": "Security",
+      "method": "GET",
+      "humanName": "get components for a host",
+      "requestUrl": "/beta/security/threatIntelligence/hosts/{hostId}/components",
+      "docLink": "https://github.com/microsoftgraph/microsoft-graph-docs/pull/20472/files?short_path=bbfa479",
+      "skipTest": false
+    },
+    {
+      "id": "2920f221-5b7a-44f8-a676-2bfb5dec85f8",
+      "category": "Security",
+      "method": "GET",
+      "humanName": "get cookies for a host",
+      "requestUrl": "/beta/security/threatIntelligence/hosts/{hostId}/cookies",
+      "docLink": "https://github.com/microsoftgraph/microsoft-graph-docs/pull/20472/files?short_path=bbfa479",
+      "skipTest": false
+    },
+    {
+      "id": "f2588a6c-289d-4822-a390-c0ee6b87bda6",
+      "category": "Security",
+      "method": "GET",
+      "humanName": "get trackers for a host",
+      "requestUrl": "/beta/security/threatIntelligence/hosts/{hostId}/trackers",
+      "docLink": "https://github.com/microsoftgraph/microsoft-graph-docs/pull/20472/files?short_path=bbfa479",
+      "skipTest": false
+    },
+    {
+      "id": "29acebc1-b91a-4971-9cf8-bf89e4d32e4d",
+      "category": "Security",
+      "method": "GET",
+      "humanName": "get the passive dns records for a host",
```

Comment:
Maybe remove the word `the` from this entry and the one below it to be consistent with other navigation properties.

Reply:
Same thing for the reputation entry?